scripts/setup-integration.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+# First this script will deploy
+REPO_ROOT=$(dirname "$0")/..
+INTEGRATION_TESTS_FOLDER=$REPO_ROOT/tests
+EXTRA_ARGS=$*
+
+sh $INTEGRATION_TESTS_FOLDER/bin/tests.sh
```

Comment:
that will work as long as we don't add locksmith etc to the `bin/tests.sh` script - which will change as soon as we have e2e tests.

I was mentioning the other day having a `scripts/run-stack.sh` that could be used for subgraph by both integration and unlock-js - as well as e2e later on.

We can leave it here for now and make these changes in another PR maybe?

Reply:
Let me rename it to run-stack.sh - will leave rest as is. We can modify it to run stack only and move away from that in the tests.sh